src/EditorFeatures/Core/EditorConfigSettings/Data/CodeStyle/CodeStyleSetting.BooleanCodeStyleSetting.cs

```diff
@@ -57,6 +58,29 @@ protected override CodeStyleOption2<bool> GetOption()
                 => _editorConfigOptions.TryGetEditorConfigOption(_option, out CodeStyleOption2<bool>? value) && value is not null
                     ? value
                     : _visualStudioOptions.GetOption(_option);
+
+            public override string? GetSettingName()
+            {
+                // Could change to op.GetType().Name == "EditorConfigStorageLocation`1" but not sure if it applies for all editorconfig settings
+                var option = _option.StorageLocations.FirstOrDefault(op => op.GetType().Name != "RoamingProfileStorageLocation");
```

Comment:
this is a bit odd.  why are we doign a reflection check here.  can you clarify why this is needed?